### DISCUSSION TITLE
What software is recomended?

### DISCUSSION THREAD

**Discussion by ecxod** [IJHack/QtPass#595] [General]
What software is recommended for development? QT Creator?

**Comment by annejan** [comment]
For larger changes, especially to the UI QtCreator is preferred.
But for any other kind of change vim, VS code or CLion are just as nice.

**Comment by ecxod** [comment]
Looks like Qt Creator is not running on my computer :))  
Looks like it also expects Windows10, which unfortunately I can not meet.  
I have to see if I can find an older version that also runs under Windows7.  

![image](https://user-images.githubusercontent.com/102165928/181995162-b6ce932a-49ca-4b83-b84a-58a724bd1d3c.png)
